Phone version handler and two little bits

bldewolf opened the pull request:
Not sure if the python2 thing will be broken on OS X or elsewhere, but it ends up using python 3 on my machine otherwise.

The phone version endpoint handler doesn't guess the current OS or anything because if you say you're anything but android you don't get music player messages anymore.  I guess we're all android for now!

bldewolf committed:
Set interpreter to python2

On systems that have both python 2 and 3, just calling python isn't good
enough.

bldewolf committed:
Move DEBUG_PROTOCOL section for received messages

Move the debug protocol output to the function where we can produce the entire
received message instead of just the payload.

bldewolf committed:
Add PHONE_VERSION endpoint handler

Add a phone version endpoint handler so that we respond appropriately when
connecting to a pebble.  The pebble sends this query at the start of an RFCOMM
connection and will assume a bare minimum of supported features if we don't
respond.

The only feature this controls (that I can tell) is whether or not the music
app sends next/previous/pause commands back to us.  Soo...this fixes the music
app so that it actually works.  Yay!

EnJens commented on pebble/pebble.py:
```diff
@@ -558,6 +554,44 @@ def _version_response(self, endpoint, data):
+			"WINDOWS" : 5,
+		}
+		# Magic prefix that the app adds
+		prefix = "\x01\xff\xff\xff\xff"
```
Comparing to the pebble app's source, I'd by far prefer the following:

```
pack('!bi', 1, -1)
```

It makes it clear the magic value consists of two values rather than a single string - hopefully we can get Team Pebble to tell us what/why those values are there :)

bldewolf committed:
Add phone control methods

I wrote these a long time ago but never pushed them.  This lets libpebble shoot
phone control messages at the pebble.  I remember it worked at the time...
